lib/process-services-cloud/src/lib/form/components/form-cloud.component.ts

```diff
@@ -280,9 +282,14 @@ export class FormCloudComponent extends FormBaseComponent implements OnChanges,
         }
     }
 
-    parseForm(formCloudRepresentationJSON: any): FormCloud {
+    parseForm(formCloudRepresentationJSON: any): FormModel {
         if (formCloudRepresentationJSON) {
-            const form = new FormCloud(formCloudRepresentationJSON, this.data, this.readOnly, this.formCloudService);
+            const formValues: FormValues = {};
+            (this.data || []).forEach(variable => {
+                formValues[variable.name] = variable.value;
+            });
+
+            const form = new FormModel(formCloudRepresentationJSON, formValues, this.readOnly);
```

Comment:
we are not injecting the formCloudService anymore so there is no way to subscribe on the formFieldValueChanged 
```
 if (this.formService) {
            this.formService.formFieldValueChanged.next(new FormFieldEvent(this, field));
        }
```
am I wrong?

Reply:
that service is wrong and was not used by the cloud model before. the "formCloudService" does not expose the api of FormService. Should we have real FormService injected and provided here?